01_prepare_host.sh

```diff
@@ -141,7 +141,11 @@ case "${CONTAINER_RUNTIME}" in
   *)
     ;;
 esac
-
-# pre-pull node and container images
-# shellcheck disable=SC1091
-source lib/image_prepull.sh
+# TODO (mboukhalfa) fake images
+if [[ "${NODES_PLATFORM}" == "fake" ]]; then
+  echo SKIP IMAGE PREPULL
```

Comment:
Could you edit this log to give a bit of context, like "Skipping image prepulling on fake nodes platform"?